frontend/packages/dev-console/src/components/topology/TopologyPage.scss

```diff
@@ -70,8 +69,4 @@
   &__shortcuts-button {
     margin-right: var(--pf-global--spacer--2xl);
   }
-
-  .pf-topology-container {
-    display: flex;
-  }
```

Comment:
I think we need this. Because of this change topology toolbar buttons are not visible in safari.

Reply:
I'm seeing the toolbar buttons in safari just fine.

![image](https://user-images.githubusercontent.com/11633780/80702757-8e53fc00-8aaf-11ea-9d9b-a7c3059855ae.png)

Reply:
Not able to see in safari ```Version 13.1 (15609.1.20.111.8)```
![safari](https://user-images.githubusercontent.com/2561818/80703941-55447780-8b01-11ea-9cb0-7a0053dede49.gif)

Reply:
OK, I can add it back as it has no ill effects for me in `Version 13.1 (14609.1.20.111.8)`